third_party/phantomjs/phantomjs.sh

```diff
@@ -19,11 +19,14 @@
 #         export FC_DEBUG="1024"
 
 if [[ "${RUNFILES}" == "" ]]; then
-  if [[ "$(basename "$(pwd)")" =~ \.runfiles$ ]]; then
-    RUNFILES="$(pwd)"
+  if [[ "$(pwd)" =~ ^(.*\.runfiles)(/.*)?$ ]]; then
+    RUNFILES="${BASH_REMATCH[1]}"
   else
     RUNFILES="${0}.runfiles"
   fi
+  if [ -d "${RUNFILES}/io_bazel_rules_closure" ]; then
```

Comment:
I'm so glad there's no transitive dependency fetching and that it isn't planned. It's not worth doing something unless you can do it right.